packages/mobile/src/invite/saga.ts

```diff
@@ -217,63 +103,14 @@ export function* sendInvite(
     // Store the Temp Address locally so we know which transactions were invites
     yield put(storeInviteeData(inviteDetails))
 
-    if (features.ESCROW_WITHOUT_CODE) {
-      if (amount) {
-        yield call(initiateEscrowTransfer, e164Number, amount, undefined, feeInfo)
-      }
-      yield call(Share.share, { message })
-      return
-    }
-
-    let transferReceipt: CeloTxReceipt | undefined
-    if (!features.KOMENCI) {
-      const context = newTransactionContext(TAG, 'Transfer to invite address')
-      yield put(
-        transferStableToken({
-          recipientAddress: temporaryAddress,
-          amount: INVITE_FEE,
-          comment: SENTINEL_INVITE_COMMENT,
-          context,
-          feeInfo,
-        })
-      )
-      transferReceipt = yield call(waitForTransactionWithId, context.id)
-      if (!transferReceipt) {
-        throw new Error('Transfer to invite address failed')
-      }
-      ValoraAnalytics.track(InviteEvents.invite_tx_complete, { escrowIncluded })
-      Logger.debug(TAG + '@sendInviteSaga', 'Sent money to new wallet')
-    }
-
-    // If this invitation has a payment attached to it, send the payment to the escrow.
-    if (currency === Currency.Dollar && amount) {
-      yield call(
-        initiateEscrowTransfer,
-        e164Number,
-        amount,
-        temporaryAddress,
-        feeInfo && { ...feeInfo, gas: feeInfo.gas.minus(transferReceipt?.gasUsed ?? 0) }
-      )
-    } else {
-      Logger.error(TAG, 'Currently only dollar escrow payments are allowed')
-    }
-
-    const addressToE164Number = { [temporaryAddress.toLowerCase()]: e164Number }
-    yield put(updateE164PhoneNumberAddresses({}, addressToE164Number))
-    if (features.KOMENCI) {
-      yield call(Share.share, { message })
-      ValoraAnalytics.track(InviteEvents.invite_complete, {
-        escrowIncluded,
-        amount: amount?.toString(),
-      })
```

Comment:
Looks like we should keep this. What do you think?

Reply:
Yes!! My bad, great catch